corehq/pillows/application.py

```diff
@@ -33,13 +33,14 @@ class AppPillow(AliasedElasticPillow):
     es_index = APP_INDEX
     default_mapping = APP_MAPPING
 
+    @classmethod
     @memoized
```

Comment:
Yep should be fine
On Nov 29, 2015 7:22 PM, "Simon Kelly" <notifications@github.com> wrote:

> In corehq/pillows/application.py
> <https://github.com/dimagi/commcare-hq/pull/9321#discussion_r46095136>:
>
> > @@ -33,13 +33,14 @@ class AppPillow(AliasedElasticPillow):
> >      es_index = APP_INDEX
> >      default_mapping = APP_MAPPING
> >
> > +    @classmethod
> >      @memoized
>
> Is it OK that these classmethods are memoized. Judging by my initial look
> it seems like it is but just mentioning anyway.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9321/files#r46095136>.
>